data/base-hunting.yaml

```diff
@@ -737,17 +890,6 @@ hunting_zones:
   - 10810
   - 10811
   - 10812
-  # https://elanthipedia.play.net/Armored_shalswar 300-400
```

Comment:
It was an accident I was in the middle of trying to fix it. I'm learning bare with me, sorry!